Examples/MAX32520/OTP_Dump/README.md

```diff
@@ -0,0 +1,44 @@
+## Description
+
+This example demonstrates read/write to OTP area inside device's internal memory. Write test irreversibly modifies the memory and it is disabled by default.
+
+## Required Connections
+
+### MAX32520 Evaluation Kit
+-   Connect a USB cable between the PC and the CN1 (USB TO UART) connector.
+-   Connect a debugger to J2 or J3 (JTAG/SWD).
+-   Select RX SEL and TX SEL on headers JP7 and JP8.
+-   Open an terminal application on the PC and connect to the EV kit's console UART at 115200, 8-N-1.
+
+### MAX32520FTHR Evaluation Kit
+-   Connect a USB cable between the PC and the J1 connector.
+-   Connect a debugger to J2.
+-   Open an terminal application on the PC and connect to the EV kit's console UART at 115200, 8-N-1.
+
+## Expected Output
+
+```
+***** OTP Memory Read/Write Example *****
+***** This example demonstrates how you can read/write OTP memory *****
+
+
+***** MAXIM AREA *****
```

Comment:
Update to "ADI AREA" instead? If applicable, we should probably start replacing the "Maxim" names with "ADI" or something similar.